semgrep-core/src/parsing/ast/AST_dockerfile.ml

```diff
@@ -78,6 +78,7 @@ type string_array = array_elt list bracket
    which changes the shell to an unsupported shell (i.e. not sh or bash).
 *)
 type argv_or_shell =
+  | Ellipsis of tok
```

Comment:
Can you add a (* semgrep-ext: *) comment before so we can quickly know which parts of the AST are semgrep extensions and which parts are original docker constructs?

Reply:
I forgot again. The approach I used elsewhere is to name the constructor `Semgrep_`something.